documentation/users-guide/source/reference.rst

```diff
@@ -36,16 +35,18 @@ Suites, Tests, and Benchmarks
    an assertion signals an error, the test is marked as "crashed" and
    remaining assertions are skipped.
 
-   If *expected-to-fail?* is set to ``#t`` or a function that when executed
-   returns a true value, then the test will be expected to fail. Such a failure
-   is treated as a successful test run. If the test passes rather than failing,
-   it is considered a test failure. This option has no effect on tests which
-   are *not implemented* or which have *crashed*.
-
-   *expected-to-fail-reason* is required if the test is expected to
-   fail. Normally it should reference a bug (a URL or at least a bug number).
-   If *expected-to-fail-reason* is supplied, *expected-to-fail?* may be
-   omitted because it is implied to be ``#t``.
+   *expected-to-fail?*, if supplied, is a function of no arguments that returns
```

Comment:
Why not call it `expected-to-fail-test` then?

Reply:
SGTM. Will update with that.

Reply:
I rebased this onto my github-ci branch because I needed to have the used libraries submoduled in.